src/main/java/com/iexec/worker/pubsub/StompClient.java

```diff
@@ -92,12 +109,34 @@ private void requestNewSession() {
     }
 
     /**
-     * Listen to new session request events and refresh the websocket
+     * Listen to session request events and refresh the websocket
      * connection by establishing a new STOMP session. Only one of 
      * the received requests in a fixed time interval
-     * (SESSION_REFRESH_DELAY) will be processed. We use @Scheduled
-     * to start the watcher asynchronously with an initial delay and
+     * {@code SESSION_REFRESH_DELAY} will be processed. We use
+     * {@code @Scheduled} to start the watcher asynchronously and
      * restart it in case a problem occurs.
+     * <br>
+     * 
+     * <p><b>Note:</b> the reason we use a queue is because the
+     * method {@link SessionHandler#handleTransportError()} is called,
```

Comment:
Not sure to understand `the method is called in some cases, two times to handle two different exceptions.`

Reply:
Can you check it now? Hope it is clearer!